fescpw/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
```

Comment:
This gitignore was added after you pushed, it appears? All of your node modules are showing up in your differential. Be sure to have your gitignore setup before you commit and push in the future to keep your github clean.